ui/src/main/java/com/epam/eco/kafkamanager/ui/topics/browser/ToTabularRecordsConverter.java

```diff
@@ -63,7 +64,16 @@ private static Record toTabularRecord(
                 valueTabulator.getAttributes((ConsumerRecord)consumerRecord);
         Map<String, String> headers = new HashMap<>();
         consumerRecord.headers().forEach( header -> headers.put(header.key(),new String(header.value())));
-        return new Record(consumerRecord, tabularValue, attributes, headers);
+        AvroSchema avroSchema = null;
```

Comment:
Schema resolution should be done using inheritance like other parts of record, e.g. valueTabulator.getSchema(...)